ui/mocks/handlers/singleSearch.ts

```diff
@@ -59,6 +64,10 @@ export const reservationUnitSearchHandlers = [
               nameFi: "Hertsin nuorisotalon sali",
               nameEn: "Hertsin nuorisotalon sali",
               nameSv: "Hertsin nuorisotalon sali",
+              lowestPrice: 0.0,
+              highestPrice: 20,
+              priceUnit:
+                "PER_HALF_DAY" as ReservationUnitsReservationUnitPriceUnitChoices,
```

Comment:
We have a generated typescript enum (in this case ReservationUnitsReservationUnitPriceUnitChoices.PerHalfDay). 

By using the enum values we would catch breaking api changes better, no?

Reply:
never mind, there's no difference between the two styles